twins/mocks/service.go

```diff
@@ -18,21 +17,20 @@ const (
 func NewService(tokens map[string]string) twins.Service {
 	auth := NewAuthNServiceClient(tokens)
 	twinsRepo := NewTwinRepository()
+	twinCache := NewTwinCache()
 	statesRepo := NewStateRepository()
 	idp := NewIdentityProvider()
 	subs := map[string]string{"chanID": "chanID"}
 	broker := NewBroker(subs)
-	return twins.New(broker, auth, twinsRepo, statesRepo, idp, "chanID", nil)
+	return twins.New(broker, auth, twinsRepo, twinCache, statesRepo, idp, "chanID", nil)
 }
 
 // CreateDefinition creates twin definition
-func CreateDefinition(names []string, subtopics []string) twins.Definition {
+func CreateDefinition(channels []string, subtopics []string) twins.Definition {
 	var def twins.Definition
-	for i, v := range names {
-		id, _ := uuid.New().ID()
+	for i := range channels {
 		attr := twins.Attribute{
-			Name:         v,
-			Channel:      id,
+			Channel:      channels[i],
 			Subtopic:     subtopics[i],
```

Comment:
Channels and subtopics slices have always the same size?

Reply:
Yes, the idea is, you send to arrays of the same length with names of channels and subtopics. Maybe it's not the best solution... This is only a mock.

Reply:
Yes, the idea is, you send to arrays of the same length with names of channels and subtopics. Maybe it's not the best solution... This is only a mock.